cql3/constants.hh

```diff
@@ -44,6 +46,10 @@ public:
                 m.set_cell(prefix, column, params.make_cell(*column.type, value));
             }
         }
+
+        virtual void prepare_for_raft(raft::broadcast_tables::update_query& query) const override {
+            query.new_value = expr::evaluate(*_e, query_options::DEFAULT).to_bytes();
```

Comment:
IIUC this will be an error if `_e` contains a bind marker instead of a constant. Please leave a FIXME comment that this works only for constants and bind markers are unimplemented at this point.

Also, I'm guessing that we don't have access to `update_parameters` at this point (compared to `execute`) - they only appear at execution time, and they contain e.g. values for bind markers, right?

Reply:
Also, could we move this implementation to some .cc file? I don't like the fact that we now include
```
#include "raft/broadcast_tables/experimental/lang.hh"
```
inside cql3/constants.hh, which is potentially included in many places.

There's already a cql3/constants.cc file - you could put the impl there.

Reply:
> Please leave a FIXME 

Done

> Also, I'm guessing that we don't have access to update_parameters at this point (compared to execute) - they only appear at execution time, and they contain e.g. values for bind markers, right?

Right

> There's already a cql3/constants.cc file - you could put the impl there.

Done